src/sentry/templates/sentry/bases/auth.html

```diff
@@ -3,6 +3,8 @@
 {% load crispy_forms_tags %}
 {% load i18n %}
 
+{% block sub-header %}{% endblock %}
```

Comment:
this isn't needed, but since the auth page shouldn't need this anyways lets kill it

(we might be able to do this for all modal pages)